lib/proxy_manager.js

```diff
@@ -167,15 +167,15 @@ function loadConfigFiles(cb) {
 
     // Require a '.json' file ending and reject dotfiles
     var config_files = [];
-    for (var i in files) {
-      if (path.basename(files[i])[0] === '.') {
-        continue;
-      } else if (!/^.*\.(json)$/i.test(files[i])) {
-        continue;
+    files.forEach(function(file) {
+      if (path.basename(file)[0] === '.') {
+        return;
+      } else if (!/^.*\.(json)$/i.test(file)) {
+        return;
       } else {
-        config_files.push(path.resolve(CONFIG_DIR, files[i]));
+        config_files.push(file);
```

Comment:
The bootstrap tests rely on file paths having a certain form.  Feel free to change to resolve, but the fix needs to keep the bootstrap tests working.  I took the easy way out.